queries/futures/types.ts

```diff
@@ -26,7 +26,7 @@ export type PositionDetail = {
 	profitLoss: Wei;
 };
 
-export type FuturesOrder = {
+export type FuturesPositionOrder = {
```

Comment:
Is this type only used inside the `FuturesPosition` type? Might be better to remove export if so just so it's easier to differentiate between `FuturesPositionOrder` and `FuturesOrder`